### ISSUE TITLE
Legacy Support Options (16 colors & ASCII encoding)

### ISSUE THREAD

**Issue by CraigEMP** [chubin/wttr.in#587]
First, let me say I love this app.  It produces lovely results on the command line.

I run a telnet BBS and would love the ability to curl the output of wttr.in to an ANSI file, so that said ANSI file can be viewed by users on my BBS.  However, two things are stopping me from doing that:

1. The BBS software I'm using (Synchronet, and probably most others) has no support for 256 color ANSI escape codes used in wttr.in's output.  As far as I can tell, the BBS software only supports the legacy 16 color escape codes.
2. The output encoding of wttr.in (I'm guessing) is strictly UTF-8, which also does not render properly in the BBS.  While I can set curl's encoding to ASCII, the line-drawing characters do not translate properly when using that option.

To try to illustrate both issues, I'll use the command `curl wttr.in/NYC?n -o sample.ans` to generate a sample ANS file.  When viewing the resulting sample file in the ANSI editor PablowDraw, the attached screen is what displays.

![image](https://user-images.githubusercontent.com/12660452/110215845-e569f080-7e79-11eb-8320-072c2760c17b.png)

If there were parameter options for both selecting either 16 or 256 color codes and selecting either ASCII or UTF-8 character encoding, that would produce great results that BBS sysops would love.

**Comment by zcake3000** [comment]
you can disable 256 color ANSI by using T flag.

**Comment by CraigEMP** [comment]
While that makes the output plain-text, it would still be nice to have _some_ colors instead of none at all.

It also doesn't address the encoding.

**Comment by chubin** [comment]
@CraigEMP Accepted 16-colors request.

Now about the encoding:

what what be the best solution?

1. ASCII Only (see: #432,  #113, #197, #221)
2. No frames
3. Encoding can be specified as the parameter

**Comment by CraigEMP** [comment]
> @CraigEMP Accepted 16-colors request.

Nice, thank you!


> Now about the encoding:
> 
> what what be the best solution?
> 
> 1. ASCII Only (see: #432,  #113, #197, #221)
> 2. No frames
> 3. Encoding can be specified as the parameter

Of the issues listed in your ASCII Only option, issue #432 is doable, as it's most similar to what I'm asking for, however it seemingly does not ask to keep support for the box and line drawing characters within Codepage 437, which make ANSI tables look very nice.

_(side note: we should be careful to not conflate plain-text discussed in issues #113 and #197 with ASCII encoding; ASCII != plain-text)_

The frameless display could be a decent workaround, but there are other characters that would still get interpreted incorrectly, like the degree and wind direction symbols, as well occasional dashes used for the sun's rays (another screenshot attached).

![image](https://user-images.githubusercontent.com/12660452/113527371-555bbb80-958b-11eb-989b-e2b09bba7c0b.png)

Specifying the encoding sounds ideal, if it's doable.

**Comment by chubin** [comment]
Yes, I think both is doable. I updated the issue description, added both feature requests to it

**Comment by Nikoh77** [comment]
Hello guys i am a SysOp, i am trying to implement wttr on my bbs but when i open ansi file i see this: 
![Schermata a 2023-02-10 19-34-57](https://user-images.githubusercontent.com/7826178/218170644-5b256522-8051-4404-b0e1-2ff74432d9bd.png)
How can i solve?
Thanks

**Comment by Nikoh77** [comment]
Hello, I am back, I solved with this cli command:
iconv -f UTF-8 -t CP437//TRANSLIT -c incoming.ans -o outcoming.ans
so it would be even more beautiful with some colors... ;-)
